interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -12,8 +12,6 @@
 
 #include "cling/Interpreter/InvocationOptions.h"
 
-#include "llvm/ADT/StringRef.h"
```

Comment:
I see. I think this is a future improvement somewhat outside of the scope of this PR.